.eslintrc

```diff
@@ -350,6 +350,7 @@
     // The following must be off so that we can enable
     // "@typescript-eslint/no-extra-semi".
     "no-extra-semi": "off",
+    "lines-between-class-members": "off",
```

Comment:
It was mentioned in the guidelines to keep the base rule off to avoid errors.
![Screenshot from 2021-12-30 00-55-40 (1)](https://user-images.githubusercontent.com/89375125/147696989-895b5cc8-67ab-4939-b626-73e373d5e792.png)

Reply:
Thanks for the info, @IamEzio! Can you please add a link to the rule info page (in this thread)? (This can be helpful for other reviewers)

Reply:
Yeah sure. [Link](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/lines-between-class-members.md) to the typescript-eslint documentation. Thanks!